flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/SourceStreamTaskTest.java

```diff
@@ -113,39 +107,6 @@
  */
 public class SourceStreamTaskTest extends SourceStreamTaskTestBase {
 
-    @Test
-    public void testInputEndedBeforeStopWithSavepointConfirmed() throws Exception {
```

Comment:
Can you elaborate why are you removing so many different tests in different files?